components/kyma-environment-broker/README.md

```diff
@@ -44,3 +44,7 @@ KEB binary allows you to override some configuration parameters. You can specify
 | **APP_DATABASE_PORT** | Defines the database port. | `5432` |
 | **APP_DATABASE_NAME** | Defines the database name. | `broker` |
 | **APP_DATABASE_SSL** | Specifies the SSL Mode for PostgrSQL. See all the possible values [here](https://www.postgresql.org/docs/9.1/libpq-ssl.html).  | `disable`|
+| **APP_KYMA_VERSION** | Specifies the default Kyma version. | None |
+| **APP_ENABLE_ON_DEMAND_VERSION** | Specifies whether a user can specify a Kyma version in a provisioning requst. | `false` |
```

Comment:
```suggestion
| **APP_ENABLE_ON_DEMAND_VERSION** | When true allows a user to specify a Kyma version in a provisioning request. | `false` |
```

Reply:
discussed with our TW, changed to "If set to `true`, a user can specify a Kyma version in a provisioning request."